scripts/o3de/o3de/export_project.py

```diff
@@ -136,3 +283,402 @@ def add_args(subparsers) -> None:
     export_subparser = subparsers.add_parser('export-project')
     add_parser_args(export_subparser)
 
+
+def get_asset_processor_batch_path(tools_build_path: pathlib.Path,
+                                   required: bool = False) -> pathlib.Path:
+    """
+    Get the expected path to the asset bundler tool
+
+    @param tools_build_path:    The tools (cmake) build path to locate AssetProcessorBatch
+    @param required:            If true, check if the asset bundler actually exists on file at the expected location, and raise an error if not
```

Comment:
The comment here should mention the asset processor and not bundler
```suggestion
    @param required:            If true, check if the asset processor actually exists on file at the expected location, and raise an error if not
```

Reply:
Was also wrong in the description